README.md

```diff
@@ -88,6 +88,16 @@ The following test will run automatically as you try to pull request: -Lighthous
 
```

Comment:
- [ ] OPTIONAL
Kindly include instruction on how to run your app, under the _Getting Started_ section of your ReadMe